examples/nat64/main.rs

```diff
@@ -150,28 +150,20 @@ fn nat_6to4(ethernet: Ethernet) -> Fallible<Either<Ethernet>> {
     }
 }
 
-fn install(qs: HashMap<String, PortQueue>) -> impl Pipeline {
+fn install_6_to_4(qs: HashMap<String, PortQueue>) -> impl Pipeline {
```

Comment:
name to `6to4` to stay consistent with the other name.

Reply:
oh, yeah, derp.